apps/people-app/webapp/src/styles/remove-extenssions.py

```
@@ -0,0 +1,233 @@
+
+def remove_extensions(obj):
+    """
+    Recursively remove 'extensions' fields from a nested dictionary or list.
+    """
+    if isinstance(obj, dict):
+        # Remove 'extensions' key if it exists
+        if 'extensions' in obj:
+            del obj['extensions']
+        
+        # Recursively process all values
+        for key, value in obj.items():
```

Ruff (0.15.10)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)